multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/MtaDescriptorPropertiesResolver.java

```diff
@@ -59,9 +59,10 @@ public DeploymentDescriptor resolve(DeploymentDescriptor descriptor) {
                                    .resolve();
 
         if (context.shouldReserveTemporaryRoute()) {
+            descriptor = handlerFactory.getLiveRoutesParametersResolver(descriptor, SupportedParameters.USE_LIVE_ROUTES)
```

Comment:
maybe add a comment here for why we're doing it, as the method name 'getLiveRoutesParametersResolver' doesn't explain much for why we're resolving live routes at this point. e.g. 
"// resolve placeholders first in case provided dependency is set to not provide idle routes"